javascript/src/components/search/MessageTableEntry.jsx

```diff
@@ -71,7 +95,8 @@ var MessageTableEntry = React.createClass({
                 <td colSpan={colSpanFixup}>
                     <MessageDetail message={this.props.message} inputs={this.props.inputs} streams={this.props.streams}
                                    allStreams={this.props.allStreams} allStreamsLoaded={this.props.allStreamsLoaded}
-                                   nodes={this.props.nodes} possiblyHighlight={this.possiblyHighlight}/>
+                                   nodes={this.props.nodes} possiblyHighlight={this.possiblyHighlight}
+                                   expandingAll={this.props.expandingAll}/>
```

Comment:
I would probably name the property differently, because setting it actually prevents rendering the rest of the message details.
Maybe expandAllRenderAsync?